src/index.js

```diff
@@ -6,6 +6,7 @@ import { BrowserRouter as Router} from 'react-router-dom'
 import { makeServer } from "./server";
 import {FilterContextProvider} from './context/filter-context'

```

Comment:
you can add an index.js in context folder and export all the context provider and useContext from there so you don't have to do multiple imports

Reply:
okay